app/assets/javascripts/uploadcare/widget/live.coffee

```diff
@@ -29,6 +29,11 @@ uploadcare.namespace '', (ns) ->
     initializeWidget(el)
 
   initializeWidget = (input, targetClass) ->
+    elCount = $(input).size() 
+    if elCount == 0
+        throw new Error "No DOM elements found matching selector"
```

Comment:
Regardless this is Coffescript, we are using function calls with parentheses (`Error("No DOM…")`) when it is possible.